RELEASING.md

```diff
@@ -0,0 +1,36 @@
+# Releasing Guide
+
+AX releasing workflow is semi-guided by the CI logic comprised of [GitHub actions scripts](.github/workflows/), [makefile](./Makefile), and [cosmos-release tool](./rust/release/).
+
+## General Release Steps
+
+> - Note: Explanations for the text in bold can be found in the [glossary section](#glossary).
+
+1. A release starts when a **changelog commit** is created.
```

Comment:
“Created” would probably be understood widely, including when I create the commit on my laptop — but that doesn’t start the release. I’d say it starts when a changelog is pushed to master (which regularly occurs via a PR).